migration/v0.37/prepare/60-move-issuer-ingress-class.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+HERE="$(dirname "$(readlink -f "${0}")")"
+ROOT="$(readlink -f "${HERE}/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+log_info "moving issuer ingress class keys"
+yq_move common .issuers.letsencrypt.prod.solvers[].http01.ingress.class .issuers.letsencrypt.prod.solvers[].http01.ingress.ingressClassName
+yq_move common .issuers.letsencrypt.staging.solvers[].http01.ingress.class .issuers.letsencrypt.staging.solvers[].http01.ingress.ingressClassName
+yq_move common .issuers.extraIssuers[].spec.acme.solvers[].http01.ingress.class .issuers.extraIssuers[].spec.acme.solvers[].http01.ingress.ingressClassName
```

Comment:
Have you tested these when there are multiple set? I don't think the `yq_move` really works within arrays.

Reply:
Yeah I wasn't sure if it was going to work so I tested it and found that it actually did :+1: